gslib/utils/unit_util.py

```diff
@@ -66,7 +73,7 @@ def _GenerateSuffixRegex():
 EIGHT_MIB = 8 * ONE_MIB
 TEN_MIB = 10 * ONE_MIB
 
-SECONDS_PER_DAY = 60 * 60 * 24L
+SECONDS_PER_DAY = long(60 * 60 * 24)
```

Comment:
why would `long` be even needed here?

Reply:
Not sure off the top of my head.  But, that's outside the scope of this changelist -- this is performing syntax conversions for Py 3 compatibility only :)